qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -211,11 +211,13 @@ def search(self, text, *, ignore_case='never', reverse=False,
         if reverse:
             self._flags |= QWebEnginePage.FindBackward
 
+        config.instance.set_str("scrolling.bar", "true")
```

Comment:
I'm not actually sure what the best way would be to implement this. I don't think it's a good idea to actually set the config value here, as we'll have to deal with restoring the value (I'm also not sure if this will save the config value to a file, I'm guessing not).

[This line is where the scrollbar is added](https://github.com/qutebrowser/qutebrowser/blob/master/qutebrowser/browser/shared.py#L286). Maybe we could do some sort of detection there if we are currently in a search (maybe pass an optional value in) and key off of that. If that approach is too messy, maybe we can keep doing this but properly save/restore the value (so customizations aren't lost on search exit, as I think they are now).

As for the config interface, maybe we could change `scrolling.bar` from a boolean setting to a string one with "never" "dynamic" (?) and "always". There's a [couple examples](https://github.com/qutebrowser/qutebrowser/pull/3680) in previous prs where we migrate settings

I'd wait for @The-Compiler to give some feedback before doing anything though, as he probably knows what the best way to implement this would be.

Reply:
Just to tack on to this, I would really like a setting to have the scrollbar always off, I would definitely find this behavior jarring (as the scrollbar is white, and as it shifts the page around). :P